Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: all
+all: fmt imports build
+
+.PHONY: fmt
+fmt:
+	gofmt .
```

Comment:
`go fmt ./...` will format all go files in the repo

Reply:
Updated by [60554d5](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/60554d58b1a6cf7037435eca43189d282ad92328)